validators/size.validator.js

```diff
@@ -0,0 +1,48 @@
+const Joi = require('joi');
+
+const sizeInputValidator = Joi.object({
+  name: Joi.string()
+    .min(1)
+    .max(2)
```

Comment:
please set max(3)

Reply:
from Joi.string().min(1).max(2) changed to Joi.string().min(1).max(3)